crates/bridge-exec/src/handler.rs

```diff
@@ -297,6 +297,9 @@ where
                 if let Ok(msg) = msg {
                     let payload = msg.payload();
                     let parsed_payload = borsh::from_slice::<Payload>(payload);
+                    let scope = msg.scope();
+                    let parsed_scope = borsh::from_slice::<Scope>(scope);
+                    debug!(msg=?msg, payload=?parsed_payload, scope=?parsed_scope, "ABC: got message from the L2 Client");
```

Comment:
```suggestion
                    debug!(msg=?msg, payload=?parsed_payload, scope=?parsed_scope, "got message from the L2 Client");
```

Reply:
Also fix the formatting here.  If the name of the log field is the same as the variable being used for it, you can just omit the field name and make it `?msg` (or `%msg` for `Display` print).  In any case, there should be spaces around the `=`.  Rustfmt can't understand the macro conventions so it can't automatically fix this.